client/src/Profile.js

```diff
@@ -0,0 +1,223 @@
+            type="text"
+            placeholder="Enter description:"
+            maxLength="1000"
+            value={description}
+            onChange={e => setDescription(e.target.value)}
+          ></textarea>
+          <hr />
+        </div>
+        <div className="socialsDiv">
+          <input
+            className="smallInput"
+            type="text"
+            placeholder="Enter facebok link:"
+            value={fbLink}
+            onChange={e => setFbLink(e.target.value)}
+          ></input>
+          <input
+            className="smallInput"
+            type="text"
+            placeholder="Enter instagram link:"
+            value={instaLink}
+            onChange={e => setInstaLink(e.target.value)}
+          ></input>
+        </div>
+      </div>
+    );
+  }
+
+  function renderDisplay() {
+    return (
+      <div className="blueBorder">
+        <div className="topDiv">
+          <div className="logoDiv">
+            <a href="https://www.cofed.coop/">
+              <img src={cofedlogo} className="logo" />
+            </a>
+          </div>
+          <div className="editDiv">
+            <button className="editButton" onClick={toggleEdit}>
+              Edit
+            </button>
+          </div>
+        </div>
+        <div className="ProfileTextTagsContainer">
+          <div className="picTextContainer">
+            <img className="profilePic" alt="Image" src={tina} />
+            <div className="profileTextContainer">
+              <b className="CoOpName">{name}</b>
+              <div className="CoOpLocation">{location}</div>
+              <a className="CoOpContact" href={websiteLink} target="_blank">
+                {website}
+              </a>
+              <a className="CoOpContact" href={emailLink} target="_blank">
+                {email}
+              </a>
+              <div className="CoOpContact">{phone} </div>
+            </div>
+          </div>
+          <div className="tagsContainer">
+            {tags &&
+              tags.map((text, index) => (
+                <Tag
+                  key={index}
+                  text={text}
+                  index={index}
+                  edit={edit}
+                  onDelete={handleDelete}
+                />
+              ))}
+          </div>
+        </div>
+        <div className="descriptions">
+          <hr />
+          <div className="header">Mission Statement</div>
+          <div className="profileinfo">{mission}</div>
+          <hr />
+          <div className="header">Description</div>
+          <div className="profileinfo">{description}</div>
+          <hr />
+        </div>
+        <div className="socialsDiv">
+          <a href={instaLink} target="_blank">
+            <img src={facebook} className="socialButton" />
+          </a>
+          <a href={fbLink} target="_blank">
+            <img src={instagram} className="socialButton" />
+          </a>
+        </div>
+      </div>
```

Comment:
this is a bit better but still pretty long so break this up into functions as well